code-challenges.test.js

```diff
@@ -24,31 +62,108 @@ var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saf
 
 // b) Create the function that makes the test pass.
 
+const arrShuffle = (arr) => {
+    let removeFirst = arr.shift();
+    for (i=arr.length - 1; i > 0; i--) {
+      let randomArr = Math.floor(Math.random() * (i + 1));
+      let newArr = arr[i];
+      arr[i] = arr[randomArr];
+      arr[randomArr] = newArr;
+    }
+    return arr
+  }
+
```

Comment:
And then after the function is written the test should be run so that it passess.